packages/mgt-components/src/graph/graph.groups.ts

```diff
@@ -149,7 +149,8 @@ export const findGroups = async (
 
     filterQuery = filterQuery ? `${filterQuery} and ` : '';
     for (const filter of filterGroups) {
-      batch.get(filter, `/groups?$filter=${filterQuery + filter}`, validGroupQueryScopes);
+      const fullUrl = `/groups?$filter=${filterQuery + filter}&$top=${top}`;
```

Comment:
I just learnt you can concatenate values in a template literal for a single `${}`. 🚀